bin/spack-modify-makefile-push-buildcache

```diff
@@ -45,6 +53,8 @@ match spack_version:
         install_regex = re.compile(r'\t\+\$\(SPACK\) -e .*--only-concrete --only=package --no-add /\$\(notdir \$@\) (.*)')
     case ["0", "20", _]:
```

Comment:
Would it be possible to have both 0.20.x and 0.21.x being handled in a single case, because they look the same to me, and fixing a potential bug would mean to touch it only once, not twice.

Reply:
Done